integrationTesting/tests/organize/tags/add.spec.ts

```diff
@@ -33,21 +33,31 @@ test.describe('Tag manager', () => {
       'get',
       []
     );
+
     const { log: putTagLog } = moxy.setZetkinApiMock(
       `/orgs/1/people/${ClaraZetkin.id}/tags/${ActivistTag.id}`,
       'put'
     );
 
-    await page.goto(appUri + `/organize/1/people/${ClaraZetkin.id}`);
+    const addTagButton = page.locator('text=Add tag');
+    const activist = page.locator('text=Activist');
+
+    page.goto(appUri + `/organize/1/people/${ClaraZetkin.id}`);
+    addTagButton.first().waitFor({ state: 'visible' });
 
     await page.locator('text=Add tag').click();
 
+    await activist.first().waitFor({ state: 'visible' });
+
+    // Select tag
+    await activist.first().click();
+
     moxy.setZetkinApiMock(`/orgs/1/people/${ClaraZetkin.id}/tags`, 'get', [
       ActivistTag,
     ]);
 
-    // Select tag
-    await page.click('text=Activist');
+    // Wait for the tag to appear on the page
+    await activist.nth(1).waitFor({ state: 'visible' });
```

Comment:
Same thing here with the `nth(1)`. Maybe we can find a more unique name for these tags, or we can use test IDs (or classes or something) to distinguish the _tags_ from other occurrences of the same text, if that's the problem?

Reply:
This is [replaced with `.first()` now](https://github.com/zetkin/app.zetkin.org/pull/1807/commits/b5ab00e0acb33f0b03213e74052f42852400d133#diff-6af562a25bf8e8f1572deb26c6077fed38390cc473b3a89a12a129ebac24b64dL60). Fairly common in these playwright tests to have a `first()` in these locator chain thingies by the looks of it. Also swapped out the activist tag for plays guitar to reduce likelihood of any overlap with other text elsewhere on the page.

Reply:
This is now `await playsGuitar.waitFor({ state: 'visible' });`